lib/private/Profile/ProfileManager.php

```diff
@@ -286,55 +304,56 @@ function (ILinkAction $action) use ($targetUser, $visitingUser) {
 		return $profileParameters;
 	}
 
+	/**
+	 * Return the filtered profile config containing only
+	 * the properties to be stored on the database
+	 */
+	private function filterProfileConfig(array $profileConfig): array {
```

Comment:
Better naming? Something like that or better:

```suggestion
	private function filterNotStoredProfileConfig(array $profileConfig): array {
```

Reply:
Did just that!